src/lightning/app/utilities/packaging/build_config.py

```diff
@@ -193,7 +191,7 @@ def to_dict(self) -> Dict:
         return {"__build_config__": asdict(self)}
 
     @classmethod
-    def from_dict(cls, d: Dict) -> Self:
```

Comment:
Self is correct here

Reply:
OK and could it be also the class name as string?

Reply:
nvm sorry, Self was not correct here